packages/components/stories/VirtualizedList.js

```diff
@@ -188,7 +188,7 @@ for (let i = collection.length; i < 100; i += 1) {
 
 storiesOf('Virtualized List', module)
 	.add('List > Table', () => (
-		<div style={{ height: '60vh' }} className="virtualized-list">
+		<div className="virtualized-list">
```

Comment:
I wrapped the list into <section style={{ height: '300px'}}></section>. It is unused to define a height.